subworkflows/local/intersect_statistics/main.nf

```diff
@@ -0,0 +1,74 @@
+//
+// INTERSECTION ANALYSIS OF BED FILES
+//
+
+include { BEDTOOLS_INTERSECT     } from '../../../modules/local/custom/bedtools_intersect'
+include { SVTK_VCF2BED           } from '../../../modules/nf-core/svtk/vcf2bed'
+include { BEDOPS_CONVERT2BED     } from '../../../modules/nf-core/bedops/convert2bed'
+include { TABIX_BGZIP            } from '../../../modules/nf-core/tabix/bgzip'
+
+workflow INTERSECT_STATISTICS {
+    take:
+    test             // channel: [val(meta), vcf, regions]
+    truth_regions    // channel: [truth bed]
+
+    main:
+
+    versions        = Channel.empty()
+
+    test.branch{
+            def vcf_file = it[1]
+            def regions_file = it[2]
+            vcf : vcf_file
+            regions : regions_file
+            other: false}
+            .set{test_samples}
```

Comment:
What's this layout? :grin:

Reply:
I have two types of inputs, one is vcf the other is regions bed file. This stream should branch them accordingly. First check assures if vcf file is given, second is to check if regions exist :D it is a bit messy I know. But let me know if you have something better on your mind